src/WebJobs.Extensions.DocumentDB/Trigger/CosmosDBTriggerAttribute.cs

```diff
@@ -87,5 +82,53 @@ public CosmosDBTriggerAttribute(string databaseName, string collectionName)
         /// collection.
         /// </summary>
         public int LeasesCollectionThroughput { get; set; }
+
+        /// <summary>
+        /// Optional.
+        /// Defines a prefix to be used within a Leases collection for this Trigger. Useful when sharing the same Lease collection among multiple Triggers
+        /// </summary>
+        public string LeaseCollectionPrefix { get; set; }
```

Comment:
I looked in the ScaleController, and they support this using the nameresolver. This should probably do the same.

Reply:
And by "this", I meant `LeaseCollectionPrefix` :-)

Reply:
Sounds good! Done!

Reply:
https://github.com/Azure/azure-webjobs-sdk-extensions/pull/337/commits/c5ff746b118b358332618c64f77d66024f0ad07a

Reply:
Can you add a test alongside some of those other NameResolver tests?

Reply:
Done